plane/source/docs/apms-failsafe-function.rst

```diff
@@ -43,7 +45,9 @@ being its neutral position. When you start your radio calibration on the
 mission planner, all your values will be at 1500. By moving your sticks,
 knobs and switches you will set your PWM range for each channel. The
 autopilot monitors your throttle channel and if it notices a drop lower
-than THR_FS_VALUE (Default is 950) it will go into failsafe mode.
+than :ref:`THR_FS_VALUE<THR_FS_VALUE>` (Default is 950) it will go into failsafe mode.
+
+.. note: ArduPilot can also detect if the RC Receiver becomes disconnected or dead (no PWM pulses), if the PWM values are grossly out of range (RC Reciever failure), or if the FS bit in an SBus receivers data stream is set, and will initiate a Failsafe
```

Comment:
better to use 'failsafe bit' instead of 'FS bit'

Reply:
done